.circleci/config.yml

```diff
@@ -366,27 +399,27 @@ workflows:
       - deploy_main_to_production:
           requires:
             - deploy_main_to_production_approval
-      - deploy_main_to_qa_approval:
-          type: approval
-          requires:
-            - build_main_and_push_to_ecr
-      - deploy_main_to_qa:
-          requires:
-            - deploy_main_to_qa_approval
-      - deploy_main_to_demo_approval:
-          type: approval
-          requires:
-            - build_main_and_push_to_ecr
-      - deploy_main_to_demo:
-          requires:
-            - deploy_main_to_demo_approval
+      # - deploy_main_to_qa_approval:
+      #     type: approval
+      #     requires:
+      #       - deploy_main_to_qa
+      # - deploy_main_to_qa:
+      #     requires:
+      #       - deploy_main_to_qa_approval
+      # - deploy_main_to_demo_approval:
+      #     type: approval
+      #     requires:
+      #       - deploy_main_to_demo
+      # - deploy_main_to_demo:
+      #     requires:
+      #       - deploy_main_to_demo_approval
```

Comment:
I don't think this needs to change, you can revert it back to the original code on the left.